hack/conformance.sh

```diff
@@ -13,6 +13,14 @@ export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
 
 sonobuoy_args="--wait --plugin _out/manifests/release/conformance.yaml"
 
+if [[ ! -z "$CONTAINER_TAG" ]]; then
```

Comment:
You probably had a reason to use `CONTAINER_TAG` and not `KUBEVIRT_CONTAINER_TAG` everywhere?

Reply:
I wanted to distinct between kubevirt components and tests components image tags,
but looking at this again it make no sense because they all should have the same tag.
I will change it to `KUBEVIRT_CONTAINER_TAG`.

Reply:
It make make sense to just use `DOCKER_TAG` and maybe later on rename `DOCKER_TAG` to a more general name, like the ones you are using.